src/helpers/commandHelper.ts

```diff
@@ -219,6 +219,11 @@ export class CommandEnvironmentHelper {
     exe.options.env.PATH = this.buildPathArray([
       config.ruby.pdkBinDir,
       config.ruby.pdkRubyBinDir,
+      // temporary fix for ffi 1.14.2
+      // remove when pdk 2.X comes out https://github.com/puppetlabs/puppet-vscode/issues/730
+      path.join(config.ruby.puppetBaseDir, 'private', 'git', 'mingw64', 'bin'),
```

Comment:
Do we need to guard these for Windows platforms only?  Won't stop merging though. It'll mostly be a noop on non-windows.

Reply:
Good point, it's unnecessary for those other ones. However PDK already does this on all platforms, so 🤷 .